test/DGmethods/compressible_Navier_Stokes/mms_bc.jl

```diff
@@ -57,34 +57,66 @@ end
   @inbounds abs(n[1] * u + n[2] * v + n[3] * w) + sqrt(ρinv * γ * P)
 end
 
+struct State{T}
+  arr::T
+end
+struct Flux{T}
+  arr::T
+end
+
+function Base.getproperty(state::State, sym::Symbol)
+  arr = getfield(state,:arr)
+  if sym == :ρ
+    @inbounds arr[_ρ]
+  elseif sym == :ρ𝐮
+    @inbounds SVector(arr[_U], arr[_V], arr[_W])
+  elseif sym == :𝐮
+    @inbounds SVector(arr[_U], arr[_V], arr[_W]) ./ arr[_ρ]
+  elseif sym == :E
```

Comment:
I suggest we use ``\rho e`` for ``E`` for clarity and consistency with the rest. Similarly for ``\rho q`` for humidities per unit volume etc.